x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -930,6 +942,13 @@ export const RulesList: React.FunctionComponent = () => {
     );
   };
 
+  const getRuleTagFilter = () => {
+    if (isRuleTagFilterEnabled) {
+      return [<RuleTagFilter tags={tags} selectedTags={tagsFilter} onChange={setTagsFilter} />];
+    }
+    return [];
```

Comment:
Oh yea, I'm doing this here so it lets me dynamically destructure the tag filter array in `toolsRight` array, since it's being hidden by a feature flag. Just saves some lines. 

```
const toolsRight = [
  <TypeFilter ... />
  ...getRuleTagFilter(),
  <ActionTypeFilter ... />
  etc
];
```

I could change it if it looks too weird, but this whole thing should go away once the feature flags get removed

Reply:
Nah, that's fine. Thanks for explaining!